doc/source/basics.rst

```diff
@@ -1364,6 +1364,20 @@ faster than sorting the entire Series and calling ``head(n)`` on the result.
    s.nsmallest(3)
    s.nlargest(3)
 
+.. versionadded:: 0.17.0
+
+``DataFrame`` also has the ``nlargest`` and ``nsmallest`` methods.
+
+.. ipython:: python
+
+   df = DataFrame({'a': [1, 10, 8, 11, -1],
+                   'b': list('abdce'),
+                   'c': [1.0, 2.0, np.nan, 3.0, 4.0]})
+   df.nlargest(3, 'a')
```

Comment:
maybe make the example have some dupes for 'a' (so that ['a','c'] is not obvious)